tests/test_base.py

```diff
@@ -46,9 +46,18 @@ def test_base_size(auto_container, container_runtime):
             "ppc64le": 160,
             "s390x": 125,
         }
-    assert (
+    container_size = (
         container_runtime.get_image_size(auto_container.image_url_or_id)
-        < BASE_CONTAINER_MAX_SIZE[LOCALHOST.system_info.arch] * 1024 * 1024
+        // 1024
+        // 1024
```

Comment:
You are loosing precision here, why?

```suggestion
        / 1024
        / 1024
```

Reply:
I've removed the min-boundary again as there are vastly different sizes depending on the version and I don't feel like tracking them per SP level for now.